internal/cli/command_factory.go

```diff
@@ -160,11 +161,15 @@ func (factory *CommandFactory) Run(cmd *cobra.Command) int {
 		log.Print(err)
 	} else {
 		logs := []terminal.Log{terminal.NewErrorLog(err)}
-		if e, ok := err.(Suggester); ok {
-			logs = append(logs, terminal.NewFollowupLog(terminal.MsgSuggestions, e.Suggestions()))
+
+		var suggester cli.ErrSuggester
+		if errors.As(err, &suggester) {
+			logs = append(logs, terminal.NewFollowupLog(terminal.MsgSuggestions, suggester.Suggestions()...))
```

Comment:
note: I attached a screenshot to my review comment regarding this, but I think we need to add a guard clause here to see if there any suggestions/links before printing the log message out.

ie:

```go
		if errors.As(err, &suggester) && len(suggester.Suggestions) > 0 {
			logs = append(logs, terminal.NewFollowupLog(terminal.MsgSuggestions, suggester.Suggestions()...))
...

```

Reply:
nice catch here